modules/textures/src/lib/parsers/parse-basis.ts

```diff
@@ -254,7 +254,7 @@ function transcodeKTX2Image(ktx2File, levelIndex: number, options): TextureLevel
     compressed,
 
     // Additional fields
-    // Add levelSize field.
+    levelSize: decodedSize,
```

Comment:
Is there some precedent for calling this levelSize? Does that already exist on the type or is it a new field? If a new field, should we consider other names? e.g. `byteSize`?

Reply:
This name is already used in dds output. This number is used by luma to calculate texture size. We have an issue in stats when have ktx2 textures.

Reply:
![image](https://user-images.githubusercontent.com/18549629/216083927-e5df550e-82e4-4ca0-88bd-e2973fbb7dd0.png)